### ISSUE TITLE
Maintainers Wanted

### ISSUE THREAD

**Issue by hmlendea** [hmlendea/gfn-electron#257]
Hello,

I'm unable to dedicate enough time and effort into this project anymore. And that has already been the case for some time now.
Additionaly, I don't use GFN myself anymore, as I rarely have time for video games these days, and when I do, it's usually games that run just fine locally and are not available of GFN anyway.

So, oficially, I'm looking for volunteers to help maintain this repo and hopefully fix the open issues that keep piling up as I can't address them.

**Comment by RedAuburn** [comment]
for other people, there's a new app under development: https://github.com/AstralVixen/GeForce-Infinity

**Comment by t0msk** [comment]
why 2 apps for same purpose? people should contribute into single project, btw I opened pull request

**Comment by t0msk** [comment]
@hmlendea this project is already open? I thought is is discounted, because I started developing GFN Infinity, I already pushed big update in pull request which refactors whole project.

**Comment by Cybertaco360** [comment]
The project has reopened! It's no longer discontinued as I'm currently working on some bug fixes along with compatibility issues. @hmlendea I'm not sure if they're working on something as of right now but I am still working on small fixes in the meantime though I admit not much has been done.

**Comment by t0msk** [comment]
> The project has reopened! It's no longer discontinued as I'm currently working on some bug fixes along with compatibility issues. [@hmlendea](https://github.com/hmlendea) I'm not sure if they're working on something as of right now but I am still working on small fixes in the meantime though I admit not much has been done.

ah okay, just asking because it is counter-productive to develop two same applications, I was asking in this thread that I would be maintainer on [Dec 13, 2024](https://github.com/hmlendea/gfn-electron/issues/257#issuecomment-2541863774) but got no response so I moved to GFN Infinity and there is already huge pull request from me, which adds tons of features to GFN

**Comment by Cybertaco360** [comment]
Originally I sent an email to the creator of the project about becoming maintainer and as of right now I'm working on this purely out of passion for the project. I do agree that it is counter-productive to develop two applications for the same reason. If you wish to become a maintainer/collaborator on this project, just know that you are more than welcome to write an email to @hmlendea to apply as a co-maintainer :)

**Comment by t0msk** [comment]
> Originally I sent an email to the creator of the project about becoming maintainer and as of right now I'm working on this purely out of passion for the project. I do agree that it is counter-productive to develop two applications for the same reason. If you wish to become a maintainer/collaborator on this project, just know that you are more than welcome to write an email to [@hmlendea](https://github.com/hmlendea) to apply as a co-maintainer :)

yeah, but I dont want to switch :/ because I already spent lot of time into Infinity project https://github.com/AstralVixen/GeForce-Infinity/pull/11 which will be after my pull request completely refactored, and contains more features than gfn-electron, so it would be step back for me, I wanted to help Linux gaming community as I work JS developer in real life.

**Comment by Cybertaco360** [comment]
I respect that decision and I agree that it would be a step back. Helping the Linux community is primarily also something I try my best to do and working as a JS developer you'd have much more experience and efficiency than a student like myself. Even if you believe that working on GFN-Electron is a step back, I hope you still wish every now and then out of the kindness in your heart to contribute to legacy projects such as this since these old repositories that shouldn't be finished just yet are what inspires new projects, new developers and new people. :)

**Comment by dlecan** [comment]
Join your efforts, please 🙏
